### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by kant2002**
Just general cleanup.
This `dir.props` is leftover from previous build infra.
makefiles also most likely do not used.

### CODE REVIEW

File: src/Microsoft.DotNet.Wpf/test/MultiTargeting.props [hunk 1]
```diff
@@ -4,28 +4,10 @@
...
-    <DefineConstants Condition="$(TargetFramework.Equals('net46'))">$(DefineConstants);NET46</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net461'))">$(DefineConstants);NET461</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net462'))">$(DefineConstants);NET462</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net47'))">$(DefineConstants);NET47</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net471'))">$(DefineConstants);NET471</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net472'))">$(DefineConstants);NET472</DefineConstants>
-
-    <DefineConstants Condition="$(TargetFramework.Equals('netcoreapp3.0'))">$(DefineConstants);NETCOREAPP3_0</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('netcoreapp3.1'))">$(DefineConstants);NETCOREAPP3_1</DefineConstants>
     <DefineConstants Condition="$(TargetFramework.Equals('net6.0'))">$(DefineConstants);NETCOREAPP5_0</DefineConstants>
```
Comment (RussKie) [inline src/Microsoft.DotNet.Wpf/test/MultiTargeting.props:7]: This one looks odd...

**Review by RussKie — APPROVED** [review]
:shipit:

File: src/Microsoft.DotNet.Wpf/test/MultiTargeting.props (hunk 1 above)
Comment (kant2002) [reply src/Microsoft.DotNet.Wpf/test/MultiTargeting.props:7]: I do not have whole visibility for the project so was not sure about this file. Also changing define may touch a lot of files.

Comment (RussKie) [reply src/Microsoft.DotNet.Wpf/test/MultiTargeting.props:7]: This was a general observation, more aimed at the WPF team.

**Comment by kant2002** [comment]
CLA check seems to be failing. I do not know why.

**Comment by ThomasGoulet73** [comment]
The cla problem does not seem to be related to this PR, I had the same problem in my latest PR and there's bunch of PR with the same problem in dotnet/runtime.

**Comment by kant2002** [comment]
Somebody else should do a review?

**Comment by vishalmsft** [comment]
Seems native files are moved our from native folder to lib. This causes installer issues downstream. 

**From:** runtimes\win-x86\native\
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\D3DCompiler_47_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\FXJITCompiler.pdb
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\PenImc_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\PenImc_cor3.pdb
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\PresentationNative_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\PresentationNative_cor3.pdb
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\vcruntime140d.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\wpfgfx_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\runtimes\win-x86\native\wpfgfx_cor3.pdb

**To:** lib\net7.0\
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\D3DCompiler_47_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\FXJITCompiler.pdb
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\PenImc_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\PenImc_cor3.pdb
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\PresentationNative_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\PresentationNative_cor3.pdb
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\vcruntime140d.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\wpfgfx_cor3.dll
wpf\artifacts\packaging\Debug\Microsoft.DotNet.Wpf.GitHub.Debug\lib\net7.0\wpfgfx_cor3.pdb

cc: @singhashish-wpf

**Comment by RussKie** [comment]
@kant2002 I suggest you build before and after the change and compare binlogs (under .\\artifacts\logs), and see what's causing this change.

**Comment by kant2002** [comment]
From what I see, this is comes down to old main from which I create this PR.

https://github.com/dotnet/wpf/pull/5974 this line has the fix (at least how I understand this)
I did rebase, and issue seems to be gone.

**Comment by kant2002** [comment]
@vishalmsft can you take a look again on this change?

**Review by dipeshmsft — APPROVED** [review]

**Comment by kant2002** [comment]
@dipeshmsft what else needed for this PR to be merged? Itching to make future small improvements to WPF.

**Pull request merged**